master/usage/policy/secure-metrics.md

```diff
@@ -0,0 +1,415 @@
+   label matches the selector of the network policy created in step 1.
+
+   Then, use calicoctl to apply the host endpoints with the following command.
+
+   ```shell
+   calicoctl apply -f host-endpoints.yaml
+   ```
+
+1. Create a network policy which restricts access to the {{site.nodecontainer}} prometheus metrics port.
+
+   Now let's create a network policy which limits access to the prometheus metrics port such that
+   only endpoints with the label `calico-prometheus-access: true` can access the metrics.
+
+   To do this, create a file named `calico-prometheus-policy.yaml` with the following contents.
+
+   ```yaml
+   # Allow traffic to prometheus only from sources which are
+   # labeled as such, but don't impact any other traffic.
+   apiVersion: projectcalico.org/v3
+   kind: GlobalNetworkPolicy
+   metadata:
+     name: restrict-calico-node-prometheus
+   spec:
+     # Select all {{site.prodname}} nodes.
+     selector: running-calico == "true"
+     order: 500
+     types:
+     - Ingress
+     ingress:
+     # Deny anything that tries to access the prometheus port
+     # but that doesn't match the necessary selector.
+     - action: Deny
+       protocol: TCP
+       source:
+         notSelector: calico-prometheus-access == "true"
+       destination:
+         ports:
+         - 9091
+   ```
+
+   This policy selects all endpoints which have the label `running-calico: true`, and enforces a single ingress deny rule.
+   The ingress rule denies traffic to port 9091 unless the source of traffic has the label `calico-prometheus-access: true`, meaning
+   all Calico workload endpoints, host endpoints, and global network sets which do not have the label, as well as any
+   other network endpoints unknown to Calico.
+
+   Then, use calicoctl to apply this policy.
+
+   ```shell
+   calicoctl apply -f calico-prometheus-policy.yaml
+   ```
+
+1. Apply labels to any endpoints which should have access to the metrics.
+
+   At this point, only endpoints which have the label `calico-prometheus-access: true` can reach
+   {{site.prodname}}'s prometheus metrics endpoints on each node. To grant access, simply add this label to the
+   desired endpoints.
+
+   For example, to allow access to a Kubernetes pod you can run the following command.
+
+   ```shell
+   kubectl label pod my-prometheus-pod calico-prometheus-access=true
+   ```
+
+   If you would like to grant access to a specific IP network, you
+   can create a [global network set](../../reference/calicoctl/resources/globalnetworkset) using calicoctl.
+
+   For example, you might want to grant access to your management subnets.
+
+   ```yaml
+   apiVersion: projectcalico.org/v3
+   kind: GlobalNetworkSet
+   metadata:
+     name: calico-prometheus-set
+     labels:
+       calico-prometheus-access: "true"
+   spec:
+     nets:
+     - 172.15.0.0/24
+     - 172.101.0.0/24
+   ```
+
+### Extended procedure for when Typha is enabled
```

Comment:
This heading is probably OK but in my opinion, it might flow better if you said "Additional steps" instead of "Extended procedure for when". Perhaps, "Additional steps for Typha deployments"